packages/mask/src/plugins/MaskBox/contexts/RootContext.tsx

```diff
@@ -1,11 +1,18 @@
 import type { ChainId } from '@masknet/web3-shared-evm'
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { PluginWeb3ContextProvider, PluginIDContextProvider } from '@masknet/plugin-infra/web3'
 import type { FC, PropsWithChildren } from 'react'
-import { TargetChainIdContext } from '@masknet/plugin-infra/web3-evm'
 
 interface Props {
     chainId: ChainId
 }
 
 export const RootContext: FC<PropsWithChildren<Props>> = ({ children, chainId }) => {
-    return <TargetChainIdContext.Provider initialState={chainId}>{children}</TargetChainIdContext.Provider>
+    return (
+        <PluginIDContextProvider value={NetworkPluginID.PLUGIN_EVM}>
```

Comment:
So you don't need it in the ENS plugin anymore?

Reply:
Added